# Poky renames libtool to $host_alias-libtool.
# ./$host_alias-libtool isn't created until after configure runs with libtool >= 2.2.2
# so we can't call # it at this point. We can safely assume a version is available 
# from PATH though

--- glib-2.12.10/configure.in.orig	2006-06-05 13:34:08.000000000 +0100
+++ glib-2.12.10/configure.in	2006-06-05 13:34:36.000000000 +0100
@@ -1174,7 +1174,7 @@
   G_MODULE_LDFLAGS=
 else
   export SED
-  G_MODULE_LDFLAGS=`(./libtool --config; echo eval echo \\$export_dynamic_flag_spec) | sh`
+  G_MODULE_LDFLAGS=`($host_alias-libtool --config; echo eval echo \\$export_dynamic_flag_spec) | sh`
 fi
 dnl G_MODULE_IMPL= don't reset, so cmd-line can override
 G_MODULE_NEED_USCORE=0
@@ -1265,7 +1265,7 @@
 	LDFLAGS="$LDFLAGS $G_MODULE_LDFLAGS"
 dnl *** check for OSF1/5.0 RTLD_GLOBAL brokenness
 	echo "void glib_plugin_test(void) { }" > plugin.c
-	${SHELL} ./libtool --mode=compile ${CC} -shared \
+	${SHELL} $host_alias-libtool --mode=compile ${CC} -shared \
 		-export-dynamic -o plugin.o plugin.c 2>&1 >/dev/null
 	AC_CACHE_CHECK([for RTLD_GLOBAL brokenness],
 		glib_cv_rtldglobal_broken,[
@@ -1339,7 +1339,7 @@
 
 AC_MSG_CHECKING(for the suffix of shared libraries)
 export SED
-shrext_cmds=`./libtool --config | grep '^shrext_cmds='`
+shrext_cmds=`$host_alias-libtool --config | grep '^shrext_cmds='`
 eval $shrext_cmds
 module=yes eval std_shrext=$shrext_cmds
 # chop the initial dot
